www/src/components/debounce-input.js

```diff
@@ -28,9 +28,10 @@ class DebounceInput extends Component {
 
   render() {
     const { value } = this.state
+    const { initialValue, ...inputAttrs } = this.props
```

Comment:
this fixed an error where something was accidentally getting provided to a DOM element that shouldn't.